core/src/main/java/org/owasp/dependencycheck/analyzer/NodePackageAnalyzer.java

```diff
@@ -337,6 +337,12 @@ private void processDependencies(JsonObject json, File baseDir, File rootFile,
                 final String name = entry.getKey();
                 final String version;
                 boolean optional = false;
+                boolean skipDev = getSettings().getBoolean(Settings.KEYS.ANALYZER_NODE_PACKAGE_SKIPDEV, false);
+                boolean isDev = entry.getValue().asJsonObject().getBoolean("dev", false);
+
+                if (isDev && skipDev) {
+                    return;
```

Comment:
Shouldn't this be continue instead of return?

```suggestion
                    continue;
```

Reply:
You are absolutely correct, it should be continue 🙂